src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -751,7 +751,7 @@ impl<S: StateStore, SD: ValueRowSerde> StorageTableInner<S, SD> {
         &self,
         start_epoch: u64,
         end_epoch: u64,
-    ) -> StorageResult<impl Stream<Item = StorageResult<ChangeLogRow>> + Send> {
+    ) -> StorageResult<impl Stream<Item = StorageResult<ChangeLogRow>> + Send + 'static> {
```

Comment:
Why is the lifetime 'static 🤔  I think it's fine for an experimental feature, but I wonder why.

Reply:
The stream itself is already `'static`. Providing with this extra info to help compiler avoid reporting false error.